src/search.cpp

```diff
@@ -596,19 +596,20 @@ namespace {
     int moveCount, quietCount;
 
     // Step 1. Initialize node
-    Thread* thisThread = pos.this_thread();
+    prefetch(ThisThread->materialTable[pos.material_key()]);
+    prefetch(ThisThread->pawnsTable[pos.material_key()]);
```

Comment:
@mcostalba: I tried to move back prefetch in do_move(), and got no speed-up. even a minor slowdown (-0.3%). The reason it does not belong in do_move() is because threads don't belong in position. So it makes sense to move them out of do_move() and into search instead.